History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java

```diff
@@ -295,8 +295,10 @@ public void modifyAttribInstanceTemporalEntity(String entityId, String payload,
 
 		// get original createdAt
 		String createdAt = now;
+		String instanceIdAdd = null;
+		final JsonArray jsonArray = parser.parse(historyDAO.getListAsJsonArray(entityList)).getAsJsonArray();
```

Comment:
now if parser give exception in any case, then request will be crashed which is not expected. better to write back in try block.

Reply:
done